jcl/src/java.management/share/classes/java/lang/management/LockInfo.java

```diff
@@ -145,7 +146,7 @@ public static LockInfo from(CompositeData compositeData) {
 	 * The string will hold both the name of the lock object's class and it's
 	 * identity hash code expressed as an unsigned hexadecimal. i.e.<br>
 	 * <p>
-	 * {@link #getClassName()}&nbsp;+&nbsp;&commat;&nbsp;+&nbsp;Integer.toHexString({@link #getIdentityHashCode()})
+	 * {@link #getClassName()} &nbsp;+&nbsp;&nbsp;+&nbsp;Integer.toHexString({@link #getIdentityHashCode()})
 	 * </p>
```

Comment:
The `</p>` was deleted above.  Was that accidental or was this one missed?

Reply:
In these cases build has been complaining  about close <p> tag unnecessary, while getting to understand why it should be a problem then found HTML never required the <p> tag to be closed—it was always optional. You may close your HTML tags so it looks like well-formed XHTML, but that isn't necessary. XHTML is more strict than HTML. and when I removed build was fine.